src/components/Header/Header.js

```diff
@@ -1,17 +1,38 @@
 import React from 'react';
 import { NavLink } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
 import PropTypes from 'prop-types';
 
-import { I18N } from '../../utils/constants';
-
 import style from './Header.module.css';
 
 const Header = (props) => {
+  const { t, i18n } = useTranslation();
   const { isAuth, login } = props;
+
+  const activateRu = (e) => {
+    i18n.changeLanguage('ru');
+  };
+
+  const activateEn = (e) => {
+    i18n.changeLanguage('en');
+  };
+
   return (
     <header className={style.header}>
+      <button onClick={(e) => activateRu(e)}>RU</button>
```

Comment:
RU/ENG strings are also a part of localisation.
If you're using the third-party solution for localisation, you'll want all the content be readable in any locale.
E.x. this "RU/ENG" string will be different for chinese or arabic users.

Reply:
https://ru.wikipedia.org/wiki/%D0%9A%D0%BE%D0%B4%D1%8B_%D1%8F%D0%B7%D1%8B%It 
is generally accepted to write a language code in Latin letters, regardless of which locale you are on. I will create additional translates for this, but I will not use them for now.

Reply:
As a language toggle it is usually acceptable to use language codes, you're right.
However, it not the best practice to hardcode them in your code - use a centralised place in the project for this. Additionally, language code are part of ISO specifications, so to avoid mistakes, they are usually imported from a third-party library.
And finally, take a look at wiki - they translate the languages for the people.
![image](https://user-images.githubusercontent.com/25229823/80471094-97c45380-894b-11ea-9d96-9e9999b01b07.png)